app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/nutrition/NutritionProductFragment.java

```diff
@@ -419,13 +352,84 @@ public void onClick(@NonNull View view) {
         binding.nutrimentsRecyclerView.setAdapter(adapter);
     }
 
+    private void setupNutrientItems(Activity currActivity, Nutriments nutriments) {
+        List<NutrientLevelItem> levelItemList = new ArrayList<>();
+        NutrientLevels nutrientLevels = product.getNutrientLevels();
+        NutrimentLevel fat = null;
+        NutrimentLevel saturatedFat = null;
+        NutrimentLevel sugars = null;
+        NutrimentLevel salt = null;
+
+        if (nutrientLevels != null) {
+            fat = nutrientLevels.getFat();
+            saturatedFat = nutrientLevels.getSaturatedFat();
+            sugars = nutrientLevels.getSugars();
+            salt = nutrientLevels.getSalt();
+        }
+
+        if (fat == null && salt == null && saturatedFat == null && sugars == null) {
+            binding.nutrientLevelsCardView.setVisibility(View.GONE);
+            levelItemList.add(new NutrientLevelItem("", "", "", 0));
+            binding.imageGrade.setVisibility(View.GONE);
+        } else {
+            // prefetch the uri
+            customTabActivityHelper = new CustomTabActivityHelper();
+            customTabActivityHelper.setConnectionCallback(this);
+
+            // TODO: Make it international
+            nutritionScoreUri = Uri.parse("https://fr.openfoodfacts.org/score-nutritionnel-france");
```

Comment:
Do we still use it ? If so: https://world-en.openfoodfacts.org/nutriscore

Reply:
Yup when you click on 
![immagine](https://user-images.githubusercontent.com/12072630/83400073-8bc42980-a402-11ea-95de-f1da24e934a8.png)
the blue link.

Reply:
Maybe we have a translated string. Let me check.

Reply:
I was convinced it was already international, and handled somewhere else.

Reply:
Done.